charts/testkube-operator/values.yaml

```diff
@@ -29,6 +29,47 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+webhook:
+  enabled: true
+  name: testkube-operator-webhook-admission
+  labels: {}
+  annotations: {}
+
+  patch:
+    enabled: true
+    image:
+      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+      tag: v1.3.0
+      pullPolicy: IfNotPresent
+    annotations: {}
+    podAnnotations: {}
+    labels: {}
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 2000
+      fsGroup: 2000
+    serviceAccount:
+      annotations: {}
+      name: testkube-operator-webhook-cert-mgr
+    nodeSelector:
+      kubernetes.io/os: linux
+    tolerations: []
+
+  certificate:
+    secretName: webhook-server-cert
+
+  createSecretJob:
+    resources: {}
+      # limits:
```

Comment:
remove commented block

Reply:
My idea was to keep it as an example, but I don't have a strong opinion

Reply:
ok, you can leave it as is